Test/ERC20.test.js

```diff
@@ -117,21 +101,22 @@ async function ERC20Test(contractName, tokenName, tokenSymbol, mintAmount, fauce
         });
 
         it('Test decreaseAllowance more than approve()', async () => {
-            const approveAmount = 11;
+            const approveAmount = Math.floor(Math.random() * 10) + 1;
+            const decreaseAmount = Math.floor(Math.random() * 10) + 11;
             await ercFunctions.approveTest(TestToken, users[1], users[2].address, approveAmount, true);
-            await ercFunctions.decreaseAllowanceTest(TestToken, users[1], users[2].address, approveAmount, 17, false, "ERC20: decreased allowance below zero");
+            await ercFunctions.decreaseAllowanceTest(TestToken, users[1], users[2].address, approveAmount, decreaseAmount, false, "ERC20: decreased allowance below zero");
         });
 
         it('Test increaseAllowance() after approve()', async () => {
-            const approveAmount = 1101;
+            const approveAmount = Math.floor(Math.random() * 1000) + 1;
             await ercFunctions.approveTest(TestToken, users[1], users[2].address, approveAmount, true);
             await ercFunctions.increaseAllowanceTest(TestToken, users[1], users[2].address, approveAmount, 7, true);
         });
 
         it('No balance can approve(), increaseAllowance() and decreaseAllowance()', async () => {
-            const approveAmount = 1101;
-            const increaseAmount = 7;
-            const newUsers = await createWallets(1, faucet2);
+            const approveAmount = Math.floor(Math.random() * 1000) + 1;
```

Comment:
put Math.floor(Math.random() * 1000) + 1 in a helper function like generateRandomizedNumber(min,max)